lib/rest_5.js

```diff
@@ -485,19 +485,13 @@ function* resolveResult(result, chainId, node) {
 }
 
 function* resolveResults(results, chainId, node) {
-    var count = 0;
-    var res = results;
-    while (count < 60 && res.filter(r => {return r.status === constants.PENDING}).length !== 0) {
-        res = yield getBlocResults(res.map(r => {return r.hash}), chainId, false, node);
-	yield promiseTimeout(1000);
-	count++;
-    }
-
-    if(count >= 60) {
-        throw new HttpError400('Transaction did not resolve');
+    const pred = res => !res.some(r => r.status === constants.PENDING)
+    const act = n => getBlocResults(results.map(r => r.hash), chainId, false, n)
```

Comment:
Should be `false, chainId, n`. The old version is wrong too

Reply:
Done